pkg/scheduler/plugins/sla/sla_test.go

```diff
@@ -0,0 +1,131 @@
+package sla
+
+import (
+	"reflect"
+	"testing"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"volcano.sh/apis/pkg/apis/scheduling"
+	"volcano.sh/volcano/pkg/scheduler/api"
+	"volcano.sh/volcano/pkg/scheduler/conf"
+	"volcano.sh/volcano/pkg/scheduler/framework"
+	"volcano.sh/volcano/pkg/scheduler/uthelper"
+)
+
+func TestSlaPlugin(t *testing.T) {
+	var (
+		ttt  = time.Hour
+		job1 = &api.JobInfo{
+			Name:      "job1",
+			Namespace: "default",
+			Tasks: map[api.TaskID]*api.TaskInfo{
+				"0": {
+					Name: "job1-ps-0",
+				},
+				"1": {
+					Name: "job1-ps-1",
+				},
+				"2": {
+					Name: "job1-worker-0",
+				},
+			},
+
+			PodGroup: &api.PodGroup{
+				PodGroup: scheduling.PodGroup{
+					ObjectMeta: metav1.ObjectMeta{
+						Annotations: map[string]string{},
+					},
+				},
+			},
+		}
+		job2 = &api.JobInfo{
+			Name:      "job1",
+			Namespace: "default",
+			Tasks: map[api.TaskID]*api.TaskInfo{
+				"0": {
+					Name: "job1-ps-0",
+				},
+				"1": {
+					Name: "job1-ps-1",
+				},
+			},
+			WaitingTime: &ttt,
+			PodGroup: &api.PodGroup{
+				PodGroup: scheduling.PodGroup{
+					ObjectMeta: metav1.ObjectMeta{
+						Annotations: map[string]string{},
+					},
+				},
+			},
+		}
+	)
+
+	tests := []struct {
+		name string
```

Comment:
Let's put case name in `uthelper.TestCommonStruct`.

Reply:
done